src/main/java/org/jboss/pnc/deliverablesanalyzer/CacheProvider.java

```diff
@@ -135,11 +201,51 @@ public DefaultCacheManager initCaches() throws IOException {
         return cacheManager;
     }
 
-    public void close(@Disposes DefaultCacheManager cacheManager) {
-        try {
-            cacheManager.close();
-        } catch (IOException e) {
-            LOGGER.warn("Failed to close cache manager {}", cacheManager.getCache().getName(), e);
+    /**
+     * Setup the distributed Infinispan cache manager from configs. It is assumed that the caches are already setup on
+     * the remote Infinispan server.
+     *
+     * In the future, this can be replaced by the built-in quarkus-infinispan-client once we don't use the embedded
+     * infinispan feature.
+     *
+     * @return remote cache manager setup to talk to the distributed infinispan server
+     * @throws RuntimeException If the infinispan remote server list, username, or password are not specified in the
+     *         config
+     */
+    private RemoteCacheManager setupDistributedCacheManager() {
+
+        throwRuntimeExceptionIfOptionalEmpty(infinispanRemoteServerList, "infinispan server list");
+        throwRuntimeExceptionIfOptionalEmpty(infinispanUsername, "infinispan username");
+        throwRuntimeExceptionIfOptionalEmpty(infinispanPassword, "infinispan password");
+
+        ConfigurationBuilder builder = new ConfigurationBuilder();
+
+        for (String server : infinispanRemoteServerList.get()) {
+            String[] serverPort = server.split(":");
```

Comment:
I would name `hostPort` only for consistency with `host` method in the builder.

Reply:
Sure